datamodel/initdb.d/03_dcsa_ovs_v1_0.sql

```diff
@@ -39,39 +76,64 @@ CREATE TABLE dcsa_ovs_v1_0.schedule (
     date_range text
 );
 
+-- Transport Call Related Entities
+
+-- @ToDo check as transportCall subscription might be obsolete
+    DROP TABLE IF EXISTS dcsa_ovs_v1_0.transport_call_subscription CASCADE;
+CREATE TABLE dcsa_ovs_v1_0.transport_call_subscription (
+    id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+    callback_url text NOT NULL,
+    vessel_imo_number varchar(7),
+    carrier_voyage_number varchar(50),
+    un_location_code varchar(5)
+    );
+
 DROP TABLE IF EXISTS dcsa_ovs_v1_0.transport_call CASCADE;
 CREATE TABLE dcsa_ovs_v1_0.transport_call (
     id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
-    schedule_id uuid NOT NULL,
-    carrier_service_code text,
-    vessel_imo_number varchar(7),
+	transport_call_sequence_number integer,
+	vessel_imo_number varchar(7),
```

Comment:
In IM vesselIMONumber does not exist on the transport_call - it is located on the transport class

Reply:
Awaiting discussion